### PR TITLE
Update to Scala 2.13.0

### PR DESCRIPTION
**Pull request by raboof**
Supersedes #562 

To keep attribution, let's merge #557 first

### CODE REVIEW

**Comment by raboof** [comment]
1.0.1 for Scala 2.13 is on its way to Maven Central now

File: build.sbt [hunk 1]
```diff
@@ -1,5 +1,7 @@
 ThisBuild / resolvers += Resolver.jcenterRepo
 
+resolvers in ThisBuild += Resolver.bintrayRepo("akka", "snapshots")
```
Comment (ignasi35) [inline build.sbt]: This can be removed already, correct?

Comment (ignasi35) [reply build.sbt]:
```suggestion

```

**Comment by ignasi35** [comment]
Both  #557 and this seem ready to merge :-)

**Comment by raboof** [comment]
rebased as #567

**Pull request closed without merging**